storage/sqlite.go

```diff
@@ -141,7 +141,7 @@ func (s *Sqlite) fillRuleTable(config *config.Config) error {
 				Type:      rule.Key.Type,
 				Name:      rule.Key.Name,
 				Namespace: rule.Key.Namespace,
-				Role:      api.Character_name[int32(api.Character_master)],
+				Role:      "",
```

Comment:
The `Role` of any character except `Actor`, is its character. Revert these lines.